src/lib/state/DY.Lib.State.PrivateKeys.fst

```diff
@@ -22,19 +24,18 @@ type private_key_type =
 %splice [ps_private_key_type] (gen_parser (`private_key_type))
 %splice [ps_private_key_type_is_well_formed] (gen_is_well_formed_lemma (`private_key_type))
 
-type private_key_value (bytes:Type0) {|bytes_like bytes|} = {
+[@@ with_bytes bytes]
```

Comment:
Same discussion as in PKI. Leave it generic and define shorter name for DY* bytes instance?